.travis.yml

```diff
@@ -1,10 +1,10 @@
-jdk:
-  - oraclejdk8 # openJDK crashes sometimes; tensorframes is compiled w.r.t JDK8
-
 sudo: required
 
 dist: trusty
 
+jdk:
```

Comment:
I am mainly following [tensorframes' travis file](https://github.com/databricks/tensorframes/blob/master/.travis.yml#L6). 
Otherwise we get the minor version error as the tensorframes dependency we use is compiled against java 8